example/benchmark/rm_lock.sh

```diff
@@ -0,0 +1,2 @@
+rm -rf /data2/users/lchhc/.cache/torch_extensions/py39_cu116/cpu_adam/lock
```

Comment:
You should change `/data2/users/lchhc` with `~`

Reply:
done.